pkg/antctl/antctl.go

```diff
@@ -639,6 +640,11 @@ $ antctl get podmulticaststats pod -n namespace`,
 		},
 	},
 	rawCommands: []rawCommand{
+		{
+			cobraCommand:      test.Command,
+			supportAgent:      true,
+			supportController: true,
```

Comment:
I agree, this command only makes sense when run "out-of-cluster" IMO, typically with the admin kubeconfig.

Reply:
Thanks for the clarification, I learnt alot from this

Reply:
@kanha-gupta I don't think this was resolved. Please don't mark a conversation as resolved unless you're positive you addressed it correctly, otherwise it's hard to keep track of the status of requested changes when doing extra rounds of review.

Reply:
yes, I have worked on it and created a new command group to conduct both pre and post installation checks